routes/index.js

```diff
@@ -1,8 +1,10 @@
 import { Router } from 'express';
-import userRoutes from './users';
+import authRoutes from 'routes/auth';
+import userRoutes from 'routes/users';
 
 const router = Router();
 
+router.use('/', authRoutes);
```

Comment:
rename to `/auth`

Reply:
fixed https://github.com/EasyEventV2/ee2_api/pull/2/commits/aaebc523a501eacbe985aa6ad73389ba10726aeb